src/main/content/_assets/js/docs.js

```diff
@@ -23,36 +23,65 @@ function searchDocs(){
     $('.toc-item:visible').length === 0 ? $('#noSearchResults').removeClass('no-display') : $('#noSearchResults').addClass('no-display');
 }
 
-function hideCategoriesIfEmpty(){
+function hideCategoriesIfEmpty() {
     let categories = $('.toc-item');
-    $.each(categories, function(index, category){
-        if($(category).attr('id') !== 'welcome-doc'){
+    $.each(categories, function (index, category) {
+        if ($(category).attr('id') !== 'welcome-doc') {
             let childLinks = $(category).find('.doc-title');
             let isVisible = areLinksVisible(childLinks);
             isVisible ? $(category).show() : $(category).hide();
         }
     });
 }
 
-function areLinksVisible(links){
+function areLinksVisible(links) {
     let areVisible = false;
-    $.each(links, function(index, link){
-        if($(link).css('display') !== 'none'){
+    $.each(links, function (index, link) {
+        if ($(link).css('display') !== 'none') {
             areVisible = true;
         }
     });
     return areVisible;
 }
 
-function selectDocInToc(){
+function selectDocInToc() {
     let currentHref = window.location.href;
-    let selectedFile = '/docs/ref/general' + currentHref.substring(currentHref.lastIndexOf('/'));
-    if(selectedFile !== '/docs/ref/general/docs-welcome.html'){
+    let pathName = '';
+    if (location.pathname.split('/')[2] == 'ref') {
+        pathName = '/docs'
+    }
+    else {
+        pathName = `/docs/${location.pathname.split('/')[2]}`
+    }
+    let selectedFile = `${pathName}/ref/general` + currentHref.substring(currentHref.lastIndexOf('/'));
+    if (selectedFile !== `${pathName}/ref/general/docs-welcome.html`) {
         $(`a[href$="${selectedFile}"]`).addClass('active-doc')
-        $( `a[href$="${selectedFile}"]` ).parent().parent().parent().find('.toc-category').click();
+        $(`a[href$="${selectedFile}"]`).parent().parent().parent().find('.toc-category').click();
     }
 }
 
-$(document).ready(function(){
+function getDocVersions() {
+    $.getJSON('/docs/docversions.json', function (docversions) {
+        let latest = docversions['latest'];
+        let pathName = location.pathname.split('/')[2];
+        $.each(docversions['versions'], function (i, version) {
+            if (latest === version) {
+                $('#doc-version-dropdown').append(`<a class="dropdown-item" href="/docs/">Latest - ${version}</a>`);
```

Comment:
if you look at the docs for the dropdown I bet there's an attribute like "selected"  you can use to be sure this is the one that is shown (no matter the order its appended)